src/components/DatePicker/index.js

```diff
@@ -2,58 +2,64 @@ import React, { useState } from 'react';
 
 import { KeyboardDatePicker } from '@material-ui/pickers';
 import EventIcon from '@material-ui/icons/Event';
-import { FormControl, makeStyles } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core';
 import PropTypes from 'prop-types';
 
 import { IconButton } from '../buttons/IconButton';
+import { FormControl } from '../inputs/FormControl';
 
 import { commonStyles } from '../../styles/iaso/common';
 
 const useStyles = makeStyles(theme => ({
     ...commonStyles(theme),
-    formControl: {
-        width: '100%',
-    },
     clearDateButton: {
         marginRight: theme.spacing(2),
         padding: 0,
         position: 'absolute',
         right: theme.spacing(4),
         top: 13,
     },
+    helperTextError: {
+        color: theme.palette.error.main,
+    },
 }));
 
 const DatePicker = ({
     label,
     onChange,
     currentDate,
-    hasError,
     clearMessage,
-    helperText,
+    required,
+    errors,
+    hideError,
 }) => {
     const classes = useStyles();
     const [dateError, setDateError] = useState(null);
+
+    const isOnError = errors.length > 0 || Boolean(dateError);
```

Comment:
if we don't have an error array, but have a `dateError`, do we display any text? At first glance it seems not